### DISCUSSION TITLE
multilayer prismatic cell

### DISCUSSION THREAD

**Discussion by masoodtamaddon** [pybamm-team/PyBaMM#1516] [Q&A] [answered]
Hi everyone,

I was wondering if there is a guideline for simulating a multilayer jelly roll inside a prismatic cell. This is to say how to get the correct voltage from a cell when I know the total size of the cell and the microscopic measure of each layers (electrodes and separator thickness)?

Best Regards,
/Msaood

**Comment by rtimms** [comment, marked as answer]
Hi! As an example you can take a look at the `Chen2020` parameter set which is for a cylindrical cell. This [paper](https://iopscience.iop.org/article/10.1149/1945-7111/ab9050) explains all the parameters and how they were collected. In this case the parameter “Electrode height [m]” corresponds the the height of the can and the parameter “Electrode width [m]” corresponds to the “unwound length” of the electrode so that you get the correct electrode cross sectional area. So you will need to do something similar for the prismatic cell. 

At the moment the standard models (e.g. DFN, SPM) are just 1D across a single electrode layer, so you won’t be able to get distributions of quantities around the jelly roll, but you can get things like the voltage or variables as a function of position across the layer. If you are interested in modelling distributions around the roll we have a paper [here](https://iopscience.iop.org/article/10.1149/1945-7111/aba44b) that couples pybamm with openpnm to model the electrochemical and thermal behaviour in the jelly. @TomTranter is working to make the code publicly available for this. 

We also have a preprint [here](https://arxiv.org/abs/2011.04187) where we develop some reduced order models for jelly roll and prismatic cells, and I’m hoping to get the models into pybamm over the next few months. 

Hope that helps!

**Reply by masoodtamaddon** [reply]
Thanks for the information and the papers. I believe it would solve my problem. Looking forward to the new release of Pybamm.

BR
/Masood

**Comment by ssingh-ipa** [comment]
@rtimms Are the examples available now where pybamm is used to simulate the behaviors or high capacity (such as 51Ah) prismatic cells? Based on my review of the Forum, one can either couple the electrochemical model to a lumped model (https://github.com/pybamm-team/PyBaMM/discussions/1711) or form factor is not really an issue (https://github.com/pybamm-team/PyBaMM/discussions/3037) for pybamm. Could you please suggest a way to model a 51Ah prismatic NMC cell?
